hack/tools/release/notes/process.go

```diff
@@ -101,6 +116,20 @@ func (g prEntriesProcessor) process(prs []pr) []notesEntry {
 	return entries
 }
 
+func (d dependenciesProcessor) generateDependencies() (notesEntry, error) {
+	repoURL := fmt.Sprintf("https://github.com/%s", d.repo)
+	deps, err := notes.NewDependencies().ChangesForURL(
+		repoURL, d.fromTag, d.toTag,
+	)
+	if err != nil {
+		return notesEntry{}, err
+	}
+	return notesEntry{
+		title:   deps,
+		section: release.Dependencies,
+	}, nil
```

Comment:
IMO this is a bit hacky (using the title to pass the body of the deps section)

I would vote to treat these "deps" differently than the notes entries because they are not really the same.